<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWindow</class>
 <widget class="QMainWindow" name="TeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>706</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>933</width>
    <height>706</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Панель управления преподователя</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>571</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Мои тесты</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Множества</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Линейные уравнения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадратные уравнения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дифференциальные уравнения</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="finishSession_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Завершить рабочую сессию</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>591</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>Информация о выбранном тесте:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация о тесте:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="testInformation_textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>49</y>
       <width>551</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата добавления теста:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество вопросов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Время на выполнение (минут):</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>551</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="startTest_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ПРОВЕРИТЬ ТЕСТ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Результат предыдущего тестирования:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="testResult_lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>551</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Операции с тестом</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Просмотреть статистику по тесту</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить тест</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="testDate_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberOfQuestions_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="testTime_lineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="testTime_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>460</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addNewTest_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новый тест</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userFIO_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>650</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить информацию о тесте</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>650</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Перезагрузить шаблон вопросов</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информация о программе</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
